coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -686,44 +626,6 @@ void update_net_receive_buffer(NrnThread* nt) {
     nrn_pragma_omp(taskwait)
 }
 
-void update_net_send_buffer_on_host(NrnThread* nt, NetSendBuffer_t* nsb) {
```

Comment:
Not dead: https://github.com/BlueBrain/nmodl/blob/4a67187535eb6d79d7ef74095dacd6434948a43d/src/codegen/codegen_acc_visitor.cpp#L322